.github/workflows/deploy-contract.yml

```
@@ -70,8 +72,15 @@ jobs:
           TOKEN: ${{ inputs.token }}
           DURATION: ${{ inputs.duration }}
           HARDHAT_IGNITION_CONFIRM_DEPLOYMENT: false
-      - name: Get contract address
+      - name: Get sepolia contract address
+        if: ${{ inputs.network == 'sepolia' }}
         run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:32: Double quote to prevent globbing and word splitting

(shellcheck)